app/components/dialogs/prioritize-build-modal.js

```diff
@@ -0,0 +1,104 @@
+import Component from '@ember/component';
+import { inject as service } from '@ember/service';
+import { computed } from '@ember/object';
+import eventually from 'travis/utils/eventually';
+import { reads } from '@ember/object/computed';
+import { task } from 'ember-concurrency';
+import {
+  bindKeyboardShortcuts,
+  unbindKeyboardShortcuts
+} from 'ember-keyboard-shortcuts';
+
+export default Component.extend({
+  keyboardShortcuts: {
+    'esc': 'toggleConfirmationModal'
+  },
+  flashes: service(),
+
+  didInsertElement() {
+    this._super(...arguments);
+    bindKeyboardShortcuts(this);
+  },
+
+  willDestroyElement() {
+    this._super(...arguments);
+    unbindKeyboardShortcuts(this);
+  },
+  isOpen: false,
+  options: computed(() => {
+    let arr = [
+      {
+        key: 'private1',
+        displayValue: 'item1',
+        cancelRunningJobsVal: false,
+        description: 'Place build at the top of the queue',
+        modalText: 'Place build at top of the queue',
+      },
+      {
+        key: 'private2',
+        displayValue: 'item2',
+        cancelRunningJobsVal: true,
+        description: 'Place build at the top of the queue and cancel all running jobs',
+        modalText: 'Place build at the top of the queue and cancel all running jobs',
+      }
+    ];
+    return arr;
+  }),
+
+  item: computed('type', 'job', 'build', function () {
+    let type = this.type;
+    let job = this.job;
+    let build = this.build;
+    if (type === 'job') {
+      return job;
+    } else {
+      return build;
+    }
+  }),
+
+  type: computed('job', 'build', function () {
+    let job = this.job;
+    if (job) {
+      return 'job';
+    } else {
+      return 'build';
+    }
+  }),
+
+  initialKey: '',
+  initial: computed('initialKey', 'options.@each.key', function () {
+    return this.options.findBy('key', this.initialKey);
+  }),
+  initialIndex: computed('initial', 'options.[]', function () {
+    return this.options.indexOf(this.initial);
+  }),
```

Comment:
I didn't find any use of these properties. Are they just a leftover?

Reply:
Removed.